packages/components/src/internal/components/settings/AuditSettings.tsx

```diff
@@ -0,0 +1,103 @@
+
+import { PermissionTypes } from '@labkey/api';
+
+import { RequiresPermission } from '../base/Permissions';
+import { LabelHelpTip } from '../base/LabelHelpTip';
+import {
+    getAppHomeFolderPath,
+    isAssayEnabled,
+    isDataChangeCommentRequirementFeatureEnabled,
+    isELNEnabled,
+    isWorkflowEnabled,
+} from '../../app/utils';
+import { useServerContext } from '../base/ServerContext';
+import { useAppContext } from '../../AppContext';
+
+export const AuditSettings: FC = () => {
+    const { api } = useAppContext();
+    const { container, moduleContext } = useServerContext();
+    const [isRequired, setIsRequired] = useState<boolean>(false);
+
+    useEffect(() => {
+        (async () => {
+            const settings = await api.folder.getAuditSettings(getAppHomeFolderPath(container, moduleContext));
+            setIsRequired(settings.requireUserComments);
+        })();
+    }, []);
+
+    const onDisableRequirement = useCallback(() => {
+        setIsRequired(false);
+        api.folder.setAuditCommentsRequired(false, getAppHomeFolderPath(container, moduleContext));
+    }, [api]);
+
+    const onEnableRequired = useCallback(() => {
+        setIsRequired(true);
+        api.folder.setAuditCommentsRequired(true, getAppHomeFolderPath(container, moduleContext));
+    }, [api]);
+
+    if (!isDataChangeCommentRequirementFeatureEnabled(moduleContext)) {
+        return null;
+    }
+
+    return (
+        <RequiresPermission perms={PermissionTypes.Admin}>
+            <div className="panel panel-default">
+                <div className="panel-heading">Audit Logging</div>
+                <div className="panel-body">
+                    Would you like to require that users provide a reason before completing certain actions?
+                    <LabelHelpTip>
+                        The following actions allow users to enter a reason for the action, which will be recorded in the audit
+                        log:
+                        <ul>
+                            <li>Deletion of samples and sample types</li>
+                            <li>Deletion of sources and source types</li>
+                            {isAssayEnabled(moduleContext) && <li>Delete of assay runs and assay designs</li>}
+                            {isELNEnabled(moduleContext) && <li>Deletion of notebooks</li>}
+                            {isWorkflowEnabled(moduleContext) && <li>Deletion of jobs</li>}
+                            <li>Deletion of storage units</li>
+                            <li>Updating sample amount or freeze/thaw count</li>
+                            <li>Checking samples in or out of storage</li>
+                            <li>Discarding samples from storage</li>
+                            <li>Moving data between projects</li>
+                        </ul>
+                    </LabelHelpTip>
+                    <div className="framed-input__container top-spacing">
```

Comment:
I really like the look of these "radio input boxes with descriptions" for the admin settings instead of just the "checkbox with a label". I could see us factoring this out into a reusable component and using it for more of those single checkbox admin settings.